### PR TITLE
feat-rating

### PR DESCRIPTION
**Pull request by iamsaurav28**
Screenshot (70)

### CODE REVIEW

File: stylesheets/components/rating.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@import url(/Stylesheets/main.css);
+
+.check {
+  margin-top: 1rem;
+  position: relative;
+  top: 5rem;
+  flex-direction: row;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (raman1999) [inline stylesheets/components/rating.css:14]: You can make separate class for display flex and center in utility class and directly use

**Review by raman1999 — COMMENTED** [review]
Everything looks good to me, suggested minor changes

File: stylesheets/components/rating.css (hunk 1 above)
Comment (iamsaurav28) [reply stylesheets/components/rating.css:14]: yes you are right but i have only used one time here , so  i didnt take it seperate .

File: comp-layouts/rating.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+     <meta charset="UTF-8">
+     <meta http-equiv="X-UA-Compatible" content="IE=edge">
+     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+     <script src="https://kit.fontawesome.com/c4d0bd5372.js" crossorigin="anonymous"></script>
+     <title>cards</title>
+
+     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+     <link rel="stylesheet" href="/stylesheets/main.css" />
+     <link rel="stylesheet" href="/stylesheets/components/rating.css" />
+     <title>Rating</title>
```
Comment (therajatg) [inline comp-layouts/rating.html:13]: <title>cards</title> could be removed

File: comp-layouts/rating.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    <div class="check">
+     <span class="fa fa-star tick"></span>                   
+    <span class="fa fa-star tick"></span>
+    <span class="fa fa-star tick"></span>
+    <span class="fa fa-star tick"></span>
+    <span class="fa fa-star tick"></span></div>
+
+</div>
+</body>
+</html>
```
Comment (therajatg) [inline comp-layouts/rating.html:44]: you can give proper indentation

**Review by therajatg — CHANGES_REQUESTED** [review]

File: comp-layouts/rating.html (hunk 2 above)
Comment (iamsaurav28) [reply comp-layouts/rating.html:13]: yes i have removed

File: comp-layouts/rating.html (hunk 3 above)
Comment (iamsaurav28) [reply comp-layouts/rating.html:44]: okkk

**Review by therajatg — APPROVED** [review]
Good to merge!

**Review by raman1999 — APPROVED** [review]

**Pull request merged**